src/app/services/heading.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { HeadingService } from './heading.service';
+
+describe('HeadingServiceService', () => {
```

Comment:
I guess it is a typo

Reply:
looks fine to me, its a test, it runs fine